src/tools/cephfs_mirror/ServiceDaemon.cc

```diff
@@ -68,11 +68,10 @@ int ServiceDaemon::init() {
     id = id.substr(CEPHFS_MIRROR_AUTH_ID_PREFIX.size());
   }
   std::string instance_id = stringify(m_rados->get_instance_id());
-
   std::map<std::string, std::string> service_metadata = {{"id", id},
                                                          {"instance_id", instance_id}};
-  int r = m_rados->service_daemon_register("cephfs-mirror", instance_id,
-                                           service_metadata);
+  string name = cct->_conf->name.get_id();
```

Comment:
m_cct->_conf...

However, could reuse `id` from line 66 (although we extract the name-id if it's of `cephfs-mirror.0` format etc., but, I'm ok with removing that).

nit: test also need to be fixed: qa/tasks/cephfs/test_mirroring.py:test_cephfs_mirror_service_daemon_status()